python/README.md

```diff
@@ -23,7 +23,7 @@ import rzpipe
 pipe = rzpipe.open("/bin/ls")
 pipe.cmd('aa')
 print(pipe.cmd("afl"))
-print(pipe.cmdj("aflj"))            # evaluates JSON and returns an object
-print(pipe.cmdj("ij").core.format)  # shows file format
+print(pipe.cmdj("aflj"))            # parses the JSON and returns a dict (see the lowercase j)
+print(pipe.cmdJ("ij").core.format)  # parses the JSON and returns a namedtuple (see the uppercase J)
```

Comment:
usually it's written like that `(note the uppercase J)`

Reply:
no, this is on the `cmdJ` not the actual cmd